### PR TITLE
feat: move concrete parameters from `manta-sdk` to `manta-parameters`

### PR DESCRIPTION
**Pull request by BoyuanFeng**
## Summary

Move concrete parameters from `manta-sdk` to `manta-parameters`

## Details

This PR moves (partly) the following files from [manta-sdk](https://github.com/Manta-Network/sdk) to [manta-rs](https://github.com/Manta-Network/manta-rs):

- [build.rs](https://github.com/Manta-Network/sdk/blob/main/build.rs)
- [data](https://github.com/Manta-Network/sdk/tree/main/data)
- [data.checkfile](https://github.com/Manta-Network/sdk/blob/main/data.checkfile)
- [src/lib.rs](https://github.com/Manta-Network/sdk/blob/main/src/lib.rs)
- [Cargo.toml](https://github.com/Manta-Network/sdk/blob/main/Cargo.toml)
- [README.md](https://github.com/Manta-Network/sdk/blob/main/README.md)

This PR is the first step to solving #100.

---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [x] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/manta-rs/blob/main/CHANGELOG.md) and added the appropriate `changelog` label to the PR.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.
- [x] Checked that changes and commits conform to the standards outlined in [`CONTRIBUTING.md`](https://github.com/manta-network/manta-rs/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: manta-parameters/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+is-it-maintained-issue-resolution = { repository = "Manta-Network/manta-rs" }
+is-it-maintained-open-issues = { repository = "Manta-Network/manta-rs" }
+maintenance = { status = "actively-developed" }
+
+[features]
+# Enable Standard Library
+std = []
+
+# Download Data from GitHub
+download = ["anyhow/std", "attohttpc", "std"]
```
Comment (bhgomes) [inline manta-parameters/Cargo.toml:32]: In new Rust, we can write this as:

```suggestion
# Download Data from GitHub
download = ["anyhow", "attohttpc", "std"]

# Enable Standard Library
std = ["anyhow?/std"]
```

and let's have the features sorted too.

File: manta-parameters/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+[dependencies]
+anyhow = { version = "1.0.56", optional = true, default-features = false }
```
Comment (bhgomes) [inline manta-parameters/Cargo.toml:35]:
```suggestion
anyhow = { version = "1.0.57", optional = true, default-features = false }
```

File: manta-parameters/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+[dev-dependencies]
+git2 = { version = "0.14.2", default-features = false }
+hex = { version = "0.4.3", default-features = false, features = ["std"] }
+manta-parameters = { path = ".", default-features = false, features = ["download"] }
+tempfile = { version = "3.3.0", default-features = false }
+walkdir = { version = "2.3.2", default-features = false }
+
+[build-dependencies]
+anyhow = { version = "1.0.56", default-features = false, features = ["std"] }
```
Comment (bhgomes) [inline manta-parameters/Cargo.toml:47]:
```suggestion
anyhow = { version = "1.0.57", default-features = false, features = ["std"] }
```

File: manta-parameters/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+edition = "2021"
+authors = ["Manta Network <contact@manta.network>"]
+readme = "README.md"
+license-file = "LICENSE"
+repository = "https://github.com/Manta-Network/manta-rs"
+homepage = "https://github.com/Manta-Network"
+documentation = "https://github.com/Manta-Network/manta-rs"
+categories = [""]
+keywords = [""]
+description = "Concrete Parameters and Interfaces for Manta."
```
Comment (bhgomes) [inline manta-parameters/Cargo.toml:13]:
```suggestion
description = "Concrete Parameters for Manta Protocols."
```

File: manta-parameters/src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+    };
+
+    /// GitHub Organization
+    pub const ORGANIZATION: &str = "manta-network";
+
+    /// Manta-RS GitHub Repository Name
+    pub const REPO: &str = "manta-rs";
+
+    /// Manta-Parameters GitHub Repository Name
+    pub const SUB_REPO: &str = "manta-parameters";
```
Comment (bhgomes) [inline manta-parameters/src/lib.rs:47]: I would call this `CRATE` instead.

**Review by bhgomes — CHANGES_REQUESTED** [review]
Looks good, a few comments before merge. Also, make sure to run the following in the `manta-parameters` directory to setup the LFS system:

```sh
git lfs track "*.lfs"
```

**Review by stechu — APPROVED** [review]

**Pull request merged**